docs/fundamentals/dashboard/standalone-for-nodejs.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Use the .NET Aspire dashboard with Node.js apps
+description: How to use the Aspire Dashboard in a Node.js application.
+ms.date: 01/15/2025
+ms.topic: tutorial
+ms.author: aapowell
+---
+
+# Tutorial: Use the .NET Aspire dashboard with Node.js apps
+
+The [.NET Aspire dashboard](overview.md) provides a great user experience for viewing telemetry, and is available as a standalone container image that can be used with any OpenTelemetry-enabled app. In this article, you'll learn how to:
+
+> [!div class="checklist"]
+>
+> - Start the .NET Aspire dashboard in standalone mode.
+> - Use the .NET Aspire dashboard with a Node.js app.
+
+## Prerequisites
+
+To complete this tutorial, you need the following:
+
+- [Docker](https://docs.docker.com/get-docker/) or [Podman](https://podman.io/).
+  - You can use an alternative container runtime, but the commands in this article are for Docker.
+- [Node.js 18 or higher](https://nodejs.org/en/download/package-manager) locally installed.
+- A sample application.
+
+## Sample application
+
+This tutorial uses an Express.js application to demonstrate the integration. You can either create a new Express.js app or use an existing Node.js application.
+
+To create a new Express.js application:
+
+1. Create a new directory for your application:
+
+    ```console
+    mkdir aspire-nodejs-sample
+    cd aspire-nodejs-sample
+    ```
+
+1. Initialize a new Node.js project:
+
+    ```console
+    npm init -y
+    ```
+
+1. Install Express.js:
+
+    ```console
+    npm install express
+    ```
+
+1. Create a basic Express.js application by creating an `app.js` file:
+
+    ```javascript
+    const express = require('express');
+    const app = express();
+    const port = process.env.PORT || 3000;
+
+    app.get('/', (req, res) => {
+        console.log('Received request for home page');
+        res.json({
+            message: 'Hello from Node.js!',
+            timestamp: new Date().toISOString()
+        });
+    });
+
+    app.get('/api/weather', (req, res) => {
+        const weather = [
+            { city: 'Seattle', temperature: 72, condition: 'Cloudy' },
+            { city: 'Portland', temperature: 68, condition: 'Rainy' },
+            { city: 'San Francisco', temperature: 65, condition: 'Foggy' },
+            { city: 'Los Angeles', temperature: 78, condition: 'Sunny' }
+        ];
+        
+        console.log('Received request for weather data');
+        res.json(weather);
+    });
+
+    app.listen(port, () => {
+        console.log(`Server running at http://localhost:${port}`);
+    });
+    ```
+
+1. Update the `package.json` file to include a start script:
+
+    :::code language="json" source="~/docs/fundamentals/dashboard/snippets/standalone-for-nodejs/package.json":::
```

Comment:
This is misleading, as it also shows the final resulting _package.json_ that includes the packages you tell the user to install later. Maybe just say to replace (copy and paste) the entire _package.json_ with this one.
```suggestion
1. Update the `package.json` file to include a start script:

    :::code language="json" source="snippets/standalone-for-nodejs/package.json":::
```